### ISSUE TITLE
-no_mass broken in pism-dev

### ISSUE THREAD

**Issue by tkleiner** [pism/pism#385]
Hi,
I have the feeling, that the "-no_mass" command line option is not evaluated correctly in pism-dev (babc950). A simple test is `examples/searise-antarctica/antspin-coarse.sh`. You can stop after the no_mass stage. Please add "thk" to the list of extra variables (`expackage="-extra_times ...,thk"`). 

This is the error message 
```
$v$Eh  diffusivity (overrides energy) (dt=0.00649)
S     1070.444:   29.71505   36.00000        527.67975      0.00000
$v$Eh  diffusivity (overrides energy) (dt=0.00649)
S     1070.450:   29.71508   36.00000        527.69219      0.00000
PISM ERROR: Ice thickness exceeds the height of the computational box (5000.0000 m).
            The model state was saved to 'nomass_30km_max_thickness.nc'. To continue this simulation,
            run with
            -i nomass_30km_max_thickness.nc -bootstrap -regrid_file nomass_30km_max_thickness.nc -allow_extrapolation -Lz N [other options]
            where N > 5000.0000.
```

**Comment by matthiasmengel** [comment]
I can reproduce the exact error. We branched from dev at commit 80896b36f7444f7. That time, the option still worked as expected.

**Comment by ckhroulev** [comment]
Thanks for the report! This issue should be fixed by 775bb1a. Please feel free to re-open it if you think that it is not resolved.
